shared_config/data_sources.json

```diff
@@ -100,7 +100,7 @@
     ]
   },
   {
-    "title": "US Personal Consumption Expenditures: Chain-type Price Index (% Change, 1 Year)",
+    "title": "US Personal Consumption Expenditures_ Chain-type Price Index (% Change, 1 Year)",
```

Comment:
Is this intentional?

Reply:
Only as I was getting errors on windows. I have reverted, but would suggest a different symbol instead of a colon.

Reply:
Ok I'll create a new issue for this.